server/src/main/java/org/elasticsearch/cluster/ClusterInfo.java

```diff
@@ -22,7 +22,6 @@
 import com.carrotsearch.hppc.ObjectHashSet;
 import com.carrotsearch.hppc.cursors.ObjectCursor;
 import com.carrotsearch.hppc.cursors.ObjectObjectCursor;
-
```

Comment:
NIT: revert

Reply:
Done in e50bfdda9de12d53edd35df811e4479554b02f5e